compiler_opt/tools/regalloc_trace/group_functions_lib_test.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit tests for group_functions_lib."""
+
+import os
+
+from absl.testing import absltest
+
+from compiler_opt.tools.regalloc_trace import group_functions_lib
+from compiler_opt.rl import corpus
+from compiler_opt.testing import corpus_test_utils
+
+
+class GroupFunctionsTest(absltest.TestCase):
+
+  def test_get_chunks_one_command_line(self):
+    corpus_folder = self.create_tempdir()
+    corpus.create_corpus_for_testing(
+        corpus_folder.full_path,
+        elements=[
+            corpus.ModuleSpec(name='module1', size=5, command_line=('-cc1',)),
+            corpus.ModuleSpec(name='module2', size=5, command_line=('-cc1',))
+        ])
+    corpus_chunks = group_functions_lib.get_chunks(corpus_folder.full_path, (),
+                                                   2)
+    self.assertDictEqual(
+        corpus_chunks, {
+            ('-cc1',): [[
```

Comment:
you're only sorting by size, and that's the same for both, so what guarantees deterministic order for the paths?